js/components/operatorSignIn/complete.tsx

```diff
@@ -40,3 +40,19 @@ export const Error = ({
     </>
   );
 };
+
+export const NfcLookupError = (): ReactElement => (
```

Comment:
Could we change this to `BadgeLookupError`? It's not really an NFC error, although I could be convinced.

Reply:
How about something like `BadgeSerialLookupError`? It's, like, not directly part of the NFC scan, but it's not an error with the badge, it's an error with finding the badge based on a different datapoint.

Reply:
Works for me 👍

Reply:
54d08e0